src/main/java/com/parasoft/findings/jenkins/coverage/model/Percentage.java

```diff
@@ -101,8 +104,8 @@ public static Percentage valueOf(final String stringRepresentation) {
                 return new Percentage(numerator, denominator);
             }
         }
-        catch (NumberFormatException exception) {
-            // ignore and throw a specific exception
+        catch (NumberFormatException exception) { // parasoft-suppress OWASP2021.A9.LGE "This is intentionally designed to ensure exceptions during percentage instance processing don't cause the build to fail."
```

Comment:
The violation is no longer reported when caught exceptions is logged, so we can remove the suppression "OWASP2021.A9.LGE".